--- a/ansible/roles/factorio/templates/factorio-launch.j2
+++ b/ansible/roles/factorio/templates/factorio-launch.j2
@@ -13,7 +13,7 @@
 SERVER_ADMINLIST="/factorio/config/server-adminlist.json"
 SERVER_BANLIST="/factorio/config/server-banlist.json"
 SERVER_WHITELIST="/factorio/config/server-whitelist.json"
-RCON_PW_FILE="/factorio/config/rconpw"
+RCON_PW_FILE="{{ factorio_rconpw_path }}"
 
 PORT="${FACTORIO_PORT:-34197}"
 RCON_PORT="${FACTORIO_RCON_PORT:-27015}"
